tests/crypto/host/main.c

```diff
@@ -3,12 +3,12 @@
 
 #include <assert.h>
 #include <openenclave/internal/cert.h>
+#include <openenclave/internal/crypto/sha.h>
```

Comment:
Nit: It doesn't look like this file needs anything except the tests.h headers and stdio.h, this could probably be cleaned up.

Reply:
Done